src/node_modules.cc

```diff
@@ -0,0 +1,414 @@
+#include "node_modules.h"
+#include <cstdio>
+#include "base_object-inl.h"
+#include "node_errors.h"
+#include "node_external_reference.h"
+#include "node_url.h"
+#include "util-inl.h"
+#include "v8-fast-api-calls.h"
+#include "v8-function-callback.h"
+#include "v8-primitive.h"
+#include "v8-value.h"
+#include "v8.h"
+
+#include "simdjson.h"
+
+namespace node {
+namespace modules {
+
+using v8::Array;
+using v8::Context;
+using v8::FunctionCallbackInfo;
+using v8::HandleScope;
+using v8::Isolate;
+using v8::Local;
+using v8::NewStringType;
+using v8::Object;
+using v8::ObjectTemplate;
+using v8::Primitive;
+using v8::String;
+using v8::Undefined;
+using v8::Value;
+
+#ifdef __POSIX__
+constexpr char kPathSeparator = '/';
+#else
+const char* const kPathSeparator = "\\/";
+#endif
+
+void BindingData::MemoryInfo(MemoryTracker* tracker) const {
+  // Do nothing
```

Comment:
Because `MemoryInfo` is required in `BindingData`.

```
../../src/memory_tracker.h:128:23: note: unimplemented pure virtual method 'MemoryInfoName' in 'BindingData'
  virtual const char* MemoryInfoName() const = 0;
```